Sparkle/SUApplicationInfo.m

```diff
@@ -12,6 +12,7 @@
 #import "SUBundleIcon.h"
 #import "SUHost.h"
 #import <AppKit/AppKit.h>
+#import <UniformTypeIdentifiers/UniformTypeIdentifiers.h>
```

Comment:
UniformTypeIdentifiers may need to be added in linked libraries for Sparkle framework and Installer Progress targets and as `Optional` to make sure it's weak linked so it's fine on systems less than macOS 11.

Reply:
Alright Xcode already made this weak automatically so this is not an issue, never mind!

Reply:
I was under the impression that Clang does this automatically when CLANG_MODULES_AUTOLINK is enabled (which is the case for Sparkle and Installer Progress).

`otool` indicates this as well (if I interpret this correctly).

Reply:
Yep you're right.